DiscordChatExporter.Core/Exporting/Filtering/FromMessageFilter.cs

```diff
@@ -0,0 +1,16 @@
+﻿using DiscordChatExporter.Core.Discord.Data;
+
+namespace DiscordChatExporter.Core.Exporting.Filtering
+{
+    public class FromMessageFilter : MessageFilter
+    {
+        private readonly string _value;
+
+        public FromMessageFilter(string value) => _value = value;
+
+        public override bool Filter(Message message) =>
+            _value == message.Author.Name ||
+            _value == message.Author.FullName ||
+            _value == message.Author.Id.ToString();
+    }
```

Comment:
For consistency:

```suggestion
            string.Equals(_value, message.Author.Name, StringComparison.OrdinalIgnoreCase) ||
            string.Equals(_value, message.Author.FullName, StringComparison.OrdinalIgnoreCase) ||
            string.Equals(_value, message.Author.Id.ToString(), StringComparison.OrdinalIgnoreCase);
    }
```

(may need `using System`)